web/pdf_page_view.js

```diff
@@ -129,6 +129,16 @@ class PDFPageView {
 
   #viewportMap = new WeakMap();
 
+  static #layerOrder = new Map([
+    ["canvasWrapper", 0],
+    ["textLayer", 1],
+    ["annotationLayer", 2],
+    ["annotationEditorLayer", 3],
+    ["xfaLayer", 2],
+  ]);
```

Comment:
Can't we just move this to a constant, place above, instead?
```suggestion
  const LAYER_ORDER = new Map([
    ["canvasWrapper", 0],
    ["textLayer", 1],
    ["annotationLayer", 2],
    ["annotationEditorLayer", 3],
    ["xfaLayer", 2],
  ]);
```